defaultclient.go

```diff
@@ -127,6 +128,12 @@ func (client *DefaultClient) ValidatePermission(claims *JWTClaims, requiredPermi
 	for placeholder, value := range permissionResources {
 		requiredPermission.Resource = strings.Replace(requiredPermission.Resource, placeholder, value, 1)
 	}
+
+	if client.permissionAllowed(claims.Permissions, requiredPermission) {
+		log("ValidatePermission: permission allowed to access resource")
```

Comment:
Please remove the log for the normal situation, otherwise it will be full of the useless logs

Reply:
right, will do! Thanks for the suggestion